apps/main/src/pages/lend.tsx

```diff
@@ -2,31 +2,24 @@ import type { NextPage } from 'next'
 import dynamic from 'next/dynamic'
 import { Navigate, Route, Routes } from 'react-router'
 import { ROUTE } from '@/lend/constants'
-import { i18n } from '@lingui/core'
-import { I18nProvider } from '@lingui/react'
 import { OverlayProvider } from '@react-aria/overlays'
 import delay from 'lodash/delay'
 import { useCallback, useEffect, useState } from 'react'
-import 'focus-visible'
 import { HashRouter } from 'react-router-dom'
 import { persister, queryClient } from '@ui-kit/lib/api/query-client'
 import { ThemeProvider } from '@ui-kit/shared/ui/ThemeProvider'
 import GlobalStyle from '@/lend/globalStyle'
 import Page from '@/lend/layout'
-import { dynamicActivate, initTranslation } from '@ui-kit/lib/i18n'
-import { messages as messagesEn } from '@/locales/en/messages.js'
 import networks from '@/lend/networks'
 import { getPageWidthClassName } from '@/lend/store/createLayoutSlice'
 import useStore from '@/lend/store/useStore'
 import { QueryProvider } from '@ui/QueryProvider'
 import { isMobile, removeExtraSpaces } from '@/lend/utils/helpers'
-import { getLocaleFromUrl } from '@/lend/utils/utilsRouter'
```

Comment:
Not important for this URL, but lots of locale code remaining that we can remove now in a new PR.

Reply:
sounds like a good plan